.editorconfig

```diff
@@ -0,0 +1,388 @@
+# Remove the line below if you want to inherit .editorconfig settings from higher directories
+root = true
+
+# All files
+[*]
+
+#### Core EditorConfig Options ####
+
+# Encoding
+charset = utf-8
+
+# Indentation and spacing
+tab_width = 4
+indent_size = 4
+indent_style = space
+
+# New line preferences
+end_of_line = crlf
```

Comment:
See https://github.com/dotnet/format/issues/1099, which suggests not specifying end_of_line.

I have had issues previously with `dotnet format whitespace --verify-no-changes` when running in Linux vs Windows due to that.

Reply:
You can remove it. No issues. As I said Git handles it automatically if you're using Git config defaults.

Reply:
Alright, removed in 0be0c7d4bdb649bf732e066fbb7e9f1ec49f9eef 🙂